pkg/app/sensor/app.go

```diff
@@ -29,6 +29,10 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
```

Comment:
might be good to have global consts for the artifact related directory names and files, so we can share them in multiple places instead of redefining them

Reply:
By _global consts_ you mean a separate file/package? The `defaultArtifactDirName` was defined in `artifcats.go` but not really used there. So, I figured moving it closer to the place it's actually used might be a good idea. But it's not really a global constant - it's more like a default value of a (to be introduced) flag.

Overall, though, I agree with the idea :)